estate/models/estate_property.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models, fields
+from datetime import timedelta
+
+class RealEstateProperty(models.Model):
```

Comment:
```suggestion

class RealEstateProperty(models.Model):
```
Runbot will complain that you should have 2 empty lines between imports and class signature